dev/galoy/kratos-values.yml.tmpl

```diff
@@ -0,0 +1,180 @@
+kratos:
```

Comment:
We should deduplicate the values here that are already being set from the default values

Reply:
done in [d9da3ce](https://github.com/GaloyMoney/charts/pull/2474/commits/d9da3ce7f231d17d66f385caea5d28f8c7ae86fa)